src/System.Private.Xml/src/System/Xml/Serialization/Compilation.cs

```diff
@@ -169,8 +169,8 @@ internal static Assembly LoadGeneratedAssembly(Type type, string defaultNamespac
             }*/
 
             // check to see if we loading explicit pre-generated assembly
-            IEnumerable<Attribute> attrs = type.GetTypeInfo().GetCustomAttributes(typeof(XmlSerializerAssemblyAttribute), false);
-            if (attrs.Count() == 0)
+            object[] attrs = type.GetCustomAttributes(typeof(XmlSerializerAssemblyAttribute), false);
```

Comment:
This code wouldn't build anymore as `MemberInfo.GetCustomAttributes` now returns `object[]` which is cannot implicitly be converted to `IEnumerable<>`. We used `GetTypeInfo` as a workaround when porting Xml, now I changed them back to what they used to be.